Common/Constants.cs

```diff
@@ -22,7 +23,7 @@ public static class Constants
         public const string SiteFolder = @"site";
         public const string LogFilesPath = @"LogFiles";
         public const string ApplicationLogFilesDirectory = "Application";
-        public const string TracePath = LogFilesPath + @"\kudu\trace";
+        public readonly static string TracePath = Path.Combine(LogFilesPath, "kudu", "trace");
```

Comment:
When running on mono, it doesn`t like the path separator '\', and we end up creating a folder name with "LogFiles\kudu\trace" instead of nested folders. using "Path.Combine" it will use the right separator when running on different environment.